method/index.js

```diff
@@ -19,10 +18,13 @@ module.exports = function Method (ssb, keystore, state) {
     },
     // TODO - rm patient from these?
     application: {
-      create: patient(CreateGroupApplication(ssb)),
-      get: patient(GetGroupApplication(ssb)),
-      list: patient(ListGroupApplication(ssb)),
-      accept: patient(AcceptGroupApplication(ssb))
+      create: patient(application.create),
```

Comment:
I havent seen this patient function before, what does it do?

Reply:
It's defined further down in the file.
I think it's needed because we need to ensure the keystore is ready before we try some things.
I should double check